.env

```diff
@@ -0,0 +1 @@
+MGC_Logging__LogLevel__Microsoft.Graph.Cli=Debug
```

Comment:
was it the goal to include it here? or should it be removed and excluded with gitignore?

Reply:
It's used in the sample's and the launch.json file. It allows printing logs when the `--debug` parameter is specified.